src/main/java/de/azapps/kafkabackup/common/offset/OffsetSink.java

```diff
@@ -62,6 +66,10 @@ private void syncOffsetsForGroup(String consumerGroup) throws IOException {
         for (Map.Entry<TopicPartition, OffsetAndMetadata> entry : topicOffsetsAndMetadata.entrySet()) {
             TopicPartition tp = entry.getKey();
             OffsetAndMetadata offsetAndMetadata = entry.getValue();
+            if (offsetAndMetadata == null) {
+               log.warn("OffsetAndMetadata not available, negative offset? for tp {}", tp);
+               return;
```

Comment:
Good to log this! Is this everything that we should do? Or should we abort the task here? I have not dug too much into the linked issue so please tell me ;) Thanks!

Reply:
This is everything to do, and continue the task. Its just a consumer-group without a topic, hence no offset data on it at all. Why that can happen in kafka, I havent looked into. Without kafka-backup skipping it, one has to manually delete the consumer-group, for kafka-backup to proceed.

Reply:
Ok cool. Looks good to me. If you remove the `exit()` I am happy to merge it :)

Reply:
Alright, only the npe commit is in. We would be glad if you make this as a new minor version so we can upgrade with an "offical" version.